### PR TITLE
Basic ltree query support

### LINKED ISSUE #554: `ltree` field support
I like the idea of supporting `ltree` field

To support this type of fields would be to implement the list of operators (`Table F.13. ltree Operators`), here https://github.com/prest/prest/blob/9b677c30fef3f06f62f8a22cafd55a075655bdbc/adapters/postgres/postgres.go#L1028:L1067

_Originally posted by @kalmarz in https://github.com/prest/prest/discussions/504_

### PR DESCRIPTION
**Pull request by diasbruno**
Added support for quering ltree columns. 

closes #554

### CODE REVIEW

**Comment by diasbruno** [comment]
@arxdsilva 

I'm trying to run the tests with 

```bash
go test -v ./adapters/postgres -run TestWhereByRequest
```

But I'm getting: 

```txt
go test -v ./adapters/postgres -run TestWhereByRequest
2022/12/27 22:15:33 [error] pq: SSL is not enabled on the server
FAIL	github.com/prest/prest/adapters/postgres	0.008s
FAIL
```

Not sure if the `[error]` is fatal or if it's hiding what is going on...

**Comment by diasbruno** [comment]
Appreciate any tips to run the tests locally.

File: adapters/postgres/postgres.go [hunk 1]
```diff
@@ -1296,6 +1296,10 @@ func GetQueryOperator(op string) (string, error) {
 		return "NOT LIKE", nil
 	case "nilike":
 		return "NOT ILIKE", nil
+	case "Lanc":
```
Comment (avelino) [inline adapters/postgres/postgres.go:1299]: being case-insensitive is best for use in the address bar (URL)
- from `Lanc` to `lanc`
- from `Rdesc` to `rdesc`

**Review by avelino — CHANGES_REQUESTED** [review]

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (diasbruno) [reply adapters/postgres/postgres.go:1299]: Consider this just placeholders. In the end, we can choose better names for all the options.

Comment (diasbruno) [reply adapters/postgres/postgres.go:1299]: This is far from start, I'm having problems setting up the project locally to run the tests. Appreciate tips to get it working...

**Comment by diasbruno** [comment]
Found it. It's clear that I'm not a strong reader https://github.com/prest/prest#run-locally :joy:

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (diasbruno) [reply adapters/postgres/postgres.go]: I've prefixed all the operators with `ltree` (lack of a better name). 
Let me know if you have better names for them.

**Comment by diasbruno** [comment]
These are the simplest operators regarding ltrees, probably the others are better implemented using custom queries.

**Review by avelino — CHANGES_REQUESTED** [review]
Great job, we need to document (write how you use the implemented functions)


https://docs.prestd.com/prestd/docs/api-reference/parameters/#operators
https://github.com/prest/prest/blob/main/docs/api-reference/parameters.md

**Comment by vmesel** [comment]
LGTM! 😉

**Comment by diasbruno** [comment]
Done. Added the parameters to the docs. 

Let me know if there is something else to add...

**Review by avelino — APPROVED** [review]
LGTM

**Comment by avelino** [comment]
Thanks @diasbruno

**Pull request merged**